tests/helpers/test_service.py

```diff
@@ -320,14 +320,20 @@ def test_async_get_all_descriptions(hass):
 
 async def test_call_with_sync_attr(hass, mock_entities):
     """Test invoking sync service calls."""
-    mock_entities["light.kitchen"].sync_method = Mock()
+    mock_method = mock_entities["light.kitchen"].sync_method = Mock()
     await service.entity_service_call(
         hass,
         [Mock(entities=mock_entities)],
         "sync_method",
-        ha.ServiceCall("test_domain", "test_service", {"entity_id": "light.kitchen"}),
+        ha.ServiceCall(
+            "test_domain",
+            "test_service",
+            {"entity_id": "light.kitchen", "area_id": "abcd"},
+        ),
     )
-    assert mock_entities["light.kitchen"].sync_method.call_count == 1
+    assert mock_method.call_count == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4930693)